index.js

```diff
@@ -1,12 +1,120 @@
 #!/usr/bin/env node
+const path = require('path')
+const homeDir = require('os').homedir()
 const program = require('commander')
 const chalk = require('chalk')
 const ora = require('ora')
+const fs = require('fs-extra')
+const inquirer = require('inquirer')
+const { createApolloFetch } = require('apollo-fetch')
+const configDir = `${homeDir}/.config/zola`
 const wait = timeout => new Promise(resolve => setTimeout(resolve, timeout))
 const getRandom = (min, max) => Math.floor(Math.random() * max) + min
+const zolaApiUrl = 'https://api.zola.ink/graphql'
+const zolaAppUrl = 'https://app.zola.ink'
 
 program.version('0.2.0')
 
+// We need this because fetch has an issue with our SSL certs atm
+process.env['NODE_TLS_REJECT_UNAUTHORIZED'] = 0
+
+const retrieveLogin = async () => {
```

Comment:
Good thing is we can use this anywhere like this 
```
const { user, token } = await retrieveLogin()
```
To secure the command (require login). The retrieveCommand method will throw an error and exit if the user has no configuration.

Reply:
W0W